### DISCUSSION TITLE
Possible bug. Temperature refuses to go back to 1.00

### DISCUSSION THREAD

**Discussion by peterhoang** [danny-avila/LibreChat#3870] [Troubleshooting]
I'm using the latest (79f9cd5). In the same chat session, if you lower the temperature, complete the chat and try to reset it to 1.00 for the next, it seems to go back to the previous set value. I haven't tested this in a new chat session.

**Comment by danny-avila** [comment]
I intentionally did this for a better way to apply those model params soon.

it's no big deal, 1 makes it so that it uses the OpenAI/provider default. This is pretty much a necessity for Ollama, which shares the backend handling with OpenAI.

It will get resolved, it's on my list. the current trade off is better
